contracts/MetadataViews.cdc

```diff
@@ -737,4 +737,195 @@ pub contract MetadataViews {
+        // license.setDescriptionLink(link: Media(file: , mediaType: ))
+        license.setDescription("This license gives the buyer Personal Use Rights, Voting Rights, and ACE Rights")
+        return license
+    }
+
+    pub fun nlpVoteMerch(): NFTLicense {
+        let license = NFTLicense().personalUse().votingRights().merchandisingRights()
+        // license.setBadgeLink(link: Media(file: , mediaType: ))
+        // license.setDescriptionLink(link: Media(file: , mediaType: ))
+        license.setDescription("This license gives the buyer Personal Use Rights, Voting Rights, and Merch Rights")
+        return license
+    }
+
+    pub fun nlpVoteCom(): NFTLicense {
+        let license = NFTLicense().personalUse().votingRights().commercialRights()
+        // license.setBadgeLink(link: Media(file: , mediaType: ))
+        // license.setDescriptionLink(link: Media(file: , mediaType: ))
+        license.setDescription("This license gives the buyer Personal Use Rights, Voting Rights, and Comm Rights")
+        return license
+    }
+
+    pub fun nlpAceMerch(): NFTLicense {
+        let license = NFTLicense().personalUse().additionalContentExperienceRights().merchandisingRights()
+        // license.setBadgeLink(link: Media(file: , mediaType: ))
+        // license.setDescriptionLink(link: Media(file: , mediaType: ))
+        license.setDescription("This license gives the buyer Personal Use Rights, ACE Rights, and Merch Rights")
+        return license
+    }
+
+    pub fun nlpAceCom(): NFTLicense {
+        let license = NFTLicense().personalUse().additionalContentExperienceRights().commercialRights()
+        // license.setBadgeLink(link: Media(file: , mediaType: ))
+        // license.setDescriptionLink(link: Media(file: , mediaType: ))
+        license.setDescription("This license gives the buyer Personal Use Rights, ACE Rights, and Comm Rights")
+        return license
+    }
+
+    pub fun nlpUtilMerch(): NFTLicense {
+        let license = NFTLicense().personalUse().votingRights().additionalContentExperienceRights().merchandisingRights()
+        // license.setBadgeLink(link: Media(file: , mediaType: ))
+        // license.setDescriptionLink(link: Media(file: , mediaType: ))
+        license.setDescription("This license gives the buyer Personal Use Rights, Voting Rights, ACE Rights, and Merch Rights")
+        return license
+    }
+
+    pub fun nlpUtilCom(): NFTLicense {
+        let license = NFTLicense().personalUse().votingRights().additionalContentExperienceRights().commercialRights()
+        // license.setBadgeLink(link: Media(file: , mediaType: ))
+        // license.setDescriptionLink(link: Media(file: , mediaType: ))
+        license.setDescription("This license gives the buyer Personal Use Rights, Voting Rights, ACE Rights, and Comm Rights")
+        return license
+    }
+
+    pub fun nlpAce(): NFTLicense {
+        let license = NFTLicense().additionalContentExperienceRights()
```

Comment:
is missing personalUse intentional here ?

Reply:
Another good catch. I'm adding them